renderer/src/main.tsx

```diff
@@ -15,8 +16,18 @@ ReactDOM.createRoot(
   </React.StrictMode>
 )
 
-window.electron.onActivityLogged(entry => {
-  console.log('[ACTIVITY] %j', entry)
+const activities: Activity[] = []
+
+window.electron.onActivityLogged(activity => {
+  activities.push(activity)
+  if (activities.length > 100) activities.shift()
+
+  console.log('[ACTIVITIES]', activities.sort((a, b) => {
+    return a.timestamp !== b.timestamp
+      ? b.timestamp - a.timestamp
+      : a.id.localeCompare(b.id)
+  }))
```

Comment:
we could also move the sorting logic to the main thread, I don't think it makes a difference